corehq/apps/hqmedia/templates/hqmedia/partials/references.html

```diff
@@ -0,0 +1,96 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block js %}
+    <script src="{% static 'hqwebapp/js/main.js' %}"></script>
+    <!-- todo add the media player back in -->
```

Comment:
I'm not sure...but the line is almost three years old, so yeah, let's say this TODO has expired.